lib/connect/api/conversations.js

```diff
@@ -12,20 +12,20 @@ const BaseService = require('./base');
 */
 class ConversationService extends BaseService {
   /**
-   * Retrieves the id of the conversation of request.
+   * Retrieves the conversation of the conversation of instance.
    *
    * @param {string} id   The unique request identifier for which retrieve the conversationId.
    *
    * @returns {string}    Conversation ID.
    */
 
-  async getConversationIdByRequest(id) {
+  async getConversation(id) {
```

Comment:
The getConversation method returns a conversation based on the "instance_id" parameter where instance_id is the ID of the object attached to the conversation. I think a better name could be getConversationByObjectId or similar.

Reply:
agree